### DISCUSSION TITLE
SDK 1.4.0: buyer-legal-type -> eforms-buyer-legal-type

### DISCUSSION THREAD

**Discussion by duncandewhurst** [OP-TED/eForms-SDK#239] [Q&A] [answered]
In [`fields.json`](https://github.com/OP-TED/eForms-SDK/blob/1.4.0/fields/fields.json) in SDK 1.4.0, the `buyer-legal-type` codelist was renamed to `eforms-buyer-legal-type`. However, the SDK documentation still references `buyer-legal-type` in [Table 30](https://docs.ted.europa.eu/eforms/1.4.0/schema/all-in-one.html#buyerSpecificInformationTable), the XML example for [4.6.6.1.2. Buyer and service providers](https://docs.ted.europa.eu/eforms/1.4.0/schema/all-in-one.html#_buyer_and_service_providers) and possibly elsewhere (I didn't check). Should the SDK documentation be updated to reference `eforms-buyer-legal-type` instead?

Similarly, there is [no controlled vocabulary for eforms-buyer-legal-type](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/eforms-buyer-legal-type), only for [buyer-legal-type](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/buyer-legal-type). Should there be a controlled vocabulary for `eforms-buyer-legal-type`?

**Comment by YvesJo** [comment, marked as answer]
The buyer-legal-type GC file available on EU Vocabularies contains 2 deprecated values that shouldn't be used anymore. There was the need to define a tailored codelist eforms-buyer-legal-type to rapidly get rid of the two values. While this tailored list is used for code validation, the reference list still remains buyer-legal-type which should be referred in the listName attribute value.

**Reply by duncandewhurst** [reply]
Understood, thanks!
